adr/ADR-8.md

```diff
@@ -254,17 +277,27 @@ place a new message in the subject for the key with a nil body and the header `K
 This preserves history and communicate to watchers, caches and gets that a delete operation should be handled - clear cache,
 return no key error etc.
 
-#### Purging a bucket
+#### Purging a key
+
+Purge is like delete but history is not preserved. This is achieved by publishing a message in the same manner as Delete using the
+`KV-Operation: PURGE` header but adding the header `Nats-Rollup: sub` in addition.
+
+This will instruct the server to place the purge operation message in the stream and in the same RAFT update delete all messages
```

Comment:
Would remove RAFT, impl detail. Just say performed as a single server operation or something like that.